osf/models/user.py

```diff
@@ -931,9 +931,10 @@ def draft_registrations_active(self):
         """
         Active draft registrations attached to a user (user is a contributor)
         """
+
         return self.draft_registrations.filter(
-            models.Q(registered_node__isnull=True) |
-            models.Q(registered_node__is_deleted=True),
+            (models.Q(registered_node__isnull=False) | models.Q(registered_node__deleted__isnull=False)),
```

Comment:
This got flipped somehow in the chaos. We do not want to surface Drafts that have a registered_node (unless the registered_node is also deleted), so this should be `is_null=True`, as it was before.